src/App.js

```diff
@@ -0,0 +1,12 @@
+import Calculator from './components/Calculator';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
- [ ] Your  App.js component should be class-based, please update this component to use class as it is part of the project requirements 👍🏿